db/migrate/20201215191958_create_rejected_contacts.rb

```diff
@@ -0,0 +1,22 @@
+class CreateRejectedContacts < ActiveRecord::Migration[6.0]
+  def change
+    create_table :rejected_contacts do |t|
+      t.references :imported_item, null: false, foreign_key: true
+      t.string :test_and_trace_account_id
+      t.string :nhs_number
+      t.boolean :is_vulnerable
+      t.string :first_name
+      t.string :middle_names
+      t.string :surname
+      t.date :date_of_birth
+      t.text :email
+      t.string :mobile
+      t.string :telephone
+      t.string :address
+      t.string :postcode
+      t.string :needs
```

Comment:
what's `needs` column for?

Reply:
ah I see, so for an imported contact it creates an actual need, but for a reject, we just record it here? Makes sense I think. @go8soft thoughts?

Reply:
@userman123 
Because its in the import so it just captures the data that was asked to be loaded. As it is rejected we cannot create a need at this point in time

Reply:
@userman123 yes I wrote them in the review